dashboard/Dockerfile

```diff
@@ -20,9 +20,9 @@ RUN apt-get update && \
     && rm -rf /var/lib/apt/lists/*
 COPY package.json /dashboard/
 COPY yarn.lock /dashboard/
-RUN cd /dashboard && npm i yarn && npx yarn install --ignore-scripts
+RUN cd /dashboard && npx yarn install
```

Comment:
```
RUN  yarn --cwd dashboard
```
WDYT?

Reply:
What about using WORKDIR so that `/dashboard` can be removed from all commands.  Anyway, this PR is about unit tests so I would rather do related changes in a new PR.

Reply:
BTW, what is the advantage of `--cwd` argument?

Reply:
To avoid "cd /dashboard".
But WORKDIR seems the right way